examples/basic/annotations/slope.py

```diff
@@ -10,21 +10,24 @@
 
 from bokeh.models import Slope
 from bokeh.plotting import figure, show
+from bokeh.palettes import Sunset10
```

Comment:
imports are out of order:
```suggestion
from bokeh.palettes import Sunset10
from bokeh.plotting import figure, show
```